.github/workflows/test-labs.yaml

```diff
@@ -0,0 +1,78 @@
+name: Test
+on:
+  workflow_dispatch:
+    inputs:
+      kube-distro:
+        type: choice
+        description: "The type of kubernetes distribution to use, i.e minikube, kind"
```

Comment:
Cool! Note: on the long run we will enable just one option to simplify debugging.

Reply:
The problem is: we use `minikube` now, but `kind` is more performant.